CMakeLists.txt

```diff
@@ -27,5 +27,6 @@ find_package(OpenGL REQUIRED)
 add_library(Osiris SHARED ${SOURCES})
 
 target_include_directories(Osiris PRIVATE ${SDL2_INCLUDE_DIRS} ${FREETYPE_INCLUDE_DIRS})
+target_precompile_headers(Osiris PRIVATE "${CMAKE_SOURCE_DIR}/Osiris/pch.h")
```

Comment:
`target_precompile_headers()` requires CMake 3.16 so `cmake_minimum_required()` should be adjusted

Reply:
I didn't think about this. Updated to 3.16.0.